test/unit/test_ovf.c

```diff
@@ -0,0 +1,54 @@
+/* radare2 - LGPL - Copyright 2020 - pancake */
+
+#include <r_util.h>
+#include "minunit.h"
+
+int test_overflow_add(void) {
+	mu_assert_true (UT8_ADD_OVFCHK (250, 32), "ut8-add 1");
+	mu_assert_false (UT8_ADD_OVFCHK (250, 2), "ut8-add 2");
+	mu_assert_false (UT16_ADD_OVFCHK (ST16_MAX, 2), "ut16-add 2");
+	mu_assert_true (ST16_ADD_OVFCHK (ST16_MAX, 2), "st16-add 2");
+	mu_assert_true (ST16_ADD_OVFCHK (UT16_MAX, 1), "st16-add 2");
+	mu_assert_true (ST16_ADD_OVFCHK (1, ST16_MAX), "st16-add 3");
+	mu_end;
+}
+
+int test_underflow_sub(void) {
+	mu_assert_true (ST16_SUB_OVFCHK (10, 210), "st16-sub-underflow");
```

Comment:
shouldn't this be `false`?

10 - 210 = -200, which can be represented in ST16, no?

Reply:
right, this seems to be a bug

Reply:
this isn't fixed yet, is it?

Reply:
@trufae see https://github.com/radareorg/radare2/pull/17216#discussion_r454868061 . Am I missing anything?

Reply:
It is an underflow because it changes the sign